From e412ece68af9b1a4ce7309774b38ec548a2d709f Mon Sep 17 00:00:00 2001
From: Gwenole Beauchesne <gwenole.beauchesne@intel.com>
Date: Mon, 28 Apr 2014 17:44:03 +0200
Subject: [PATCH 1/6] plugins: compile the built-in video parsers as
 "vaapiparse" element.

The built-in video parsers elements are built into a single DSO named
libgstvaapi_parse.so. The various video parsers could be accessed as
vaapiparse_CODEC.
---
 gst/vaapi/gsth264parse.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/gst/vaapi/gsth264parse.c b/gst/vaapi/gsth264parse.c
index 915c2d7..96901e0 100644
--- a/gst/vaapi/gsth264parse.c
+++ b/gst/vaapi/gsth264parse.c
@@ -26,6 +26,7 @@
 #  include "config.h"
 #endif
 
+#include "gstvaapiparse.h"
 #include <gst/base/base.h>
 #include <gst/pbutils/pbutils.h>
 #include <gst/video/video.h>
@@ -122,7 +123,8 @@ gst_h264_parse_class_init (GstH264ParseClass * klass)
   GstBaseParseClass *parse_class = GST_BASE_PARSE_CLASS (klass);
   GstElementClass *gstelement_class = GST_ELEMENT_CLASS (klass);
 
-  GST_DEBUG_CATEGORY_INIT (h264_parse_debug, "h264parse", 0, "h264 parser");
+  GST_DEBUG_CATEGORY_INIT (h264_parse_debug, "vaapiparse_h264", 0,
+      "h264 parser");
 
   gobject_class->finalize = gst_h264_parse_finalize;
   gobject_class->set_property = gst_h264_parse_set_property;
-- 
2.5.1

